column-samples/custom-hover-cards/README.md

```diff
@@ -0,0 +1,42 @@
+# Custom hover card on Lists
+
+## Summary
+The following image shows a list with a custom hover added to a List:
+
+On hover - Metadata on the column "Status" is made available via column formatting
+
+![screenshot of the sample](./Status_hover.png)
+
+## View requirements
+- This format can be applied to any column type though the example is based on - 
+
+Column Name|Type
+--------|---------
+Status  | Choice
```

Comment:
The JSON is using a few more columns. Please add reference info about those as well.

Reply:
Done